src/Snap/Internal/Http/Server/Config.hs

```diff
@@ -130,6 +138,15 @@ instance Show ConfigLog where
     show (ConfigFileLog f) = "log to file " ++ show f
     show (ConfigIoLog _)   = "custom logging handler"
 
+------------------------------------------------------------------------------
+-- | This handler may be used (in conjunction with setErrorLogHandler) to write out error logs in a 
+-- custom manner.
+type ErrorLogHandler = ByteString -> IO ByteString
```

Comment:
I've just tried to work out what you meant by this comment and I can't work it out. :( I think you mean: remove the ErrorLogHandler definition and just use ByteString -> IO ByteString directly all around the code?

But part of me is thinking that maybe you want me to move these type definitions to: src/Snap/Internal/Http/Server/Types.hs

I'm just confused because the only thing that _accessLog does that is similar to this is the ConfigIoLog constructor.

So maybe what you really want me to do is:

    data ErrorLogHandler = ErrorLogHandler (ByteString -> IO ByteString)

Is that more like what you meant?